include/matx/core/tensor_utils.h

```diff
@@ -219,7 +219,7 @@ namespace detail {
   }
 
   template <typename T0, typename T1, typename... Tn>
-  constexpr auto matx_max(T0 &&t0, T1 &&t1, Tn &&... tn)
+  constexpr auto  __MATX_HOST__ __MATX_DEVICE__ matx_max(T0 &&t0, T1 &&t1, Tn &&... tn)
```

Comment:
Did you mean to keep the device qualifier here?

Reply:
Yes, for a compiler bug and to get rid of relaxed constexpr